modelling/src/main/java/org/axonframework/modelling/saga/AnnotatedSaga.java

```diff
@@ -55,12 +57,14 @@
     public AnnotatedSaga(String sagaId,
                          Set<AssociationValue> associationValues,
                          T annotatedSaga,
-                         SagaModel<T> metaModel) {
+                         SagaModel<T> metaModel,
+                         MessageHandlerInterceptorMemberChain<T> chainedInterceptor) {
```

Comment:
This new parameter should adjust the constructor's JavaDoc.

Reply:
added